packages/react-scripts/template/.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 10
+  - '10'
```

Comment:
top travis uses double quotes, this is single. Sorry for the nitpick. I don't care which one wins.

Reply:
yeah... my yaml lint fixer automatically put it to single quotes...